### ISSUE TITLE
Relative paths are not served well

### ISSUE THREAD

**Issue by doronbehar** [lomirus/live-server#95]
MWE of an HTML/CSS website layout:

```
$ tree .
.
├── html
└── my
    └── page
        ├── index.css
        └── index.html

3 directories, 3 files
```

### `my/page/index.css`

```css
body {
  direction:rtl;
}
```

### `my/page/index.html`

```html
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <link rel="stylesheet" type="text/css" href="index.css"/>
  <title>-</title>
</head>
<body>
  Hello world

</body>
</html>
```

Logs:

```
[2024-08-04T15:31:42Z WARN  live_server::server] Failed to read "/home/doron/repos/live-server-relative-paths-issue-example/index.html": No such file or directory (os error 2)
[2024-08-04T15:31:42Z WARN  live_server::server] Failed to read "/home/doron/repos/live-server-relative-paths-issue-example/favicon.ico": No such file or directory (os error 2)
[2024-08-04T15:31:47Z WARN  live_server::server] Failed to read "/home/doron/repos/live-server-relative-paths-issue-example/my/index.css": No such file or directory (os error 2)
```

Whereas if I open the same `index.html` file directly in the browser, the css file is loaded correctly - as can be seen by the `body` direction.

When working on [my website](https://gitlab.com/doronbehar/doronbehar.gitlab.io), which has multiple `index.css` files, this bug was very confusing to me.

**Comment by akx** [comment]
What is your `live-server` command line (in relation to those files) and the HTTP requests you're making?

**Comment by doronbehar** [comment]
> What is your `live-server` command line (in relation to those files)

Only `--open` is used as a command line argument, and it is launched in `/home/doron/repos/live-server-relative-paths-issue-example`.

> and the HTTP requests you're making?

I'm using the browser (Firefox).

**Comment by akx** [comment]
> and the HTTP requests you're making?

I mean, is it e.g. http://127.0.0.1:56122/my/page/index.html then?

**Comment by doronbehar** [comment]
I don't understand your question. Try to setup the directories and files as I wrote in my initial comment, and tell me if from some reason you don't experience the same 404 warnings.

In general, it is not that helpful if you ask questions that don't seem to lead into verifying a certain suspicion of the source of the bug.

**Comment by lomirus** [comment]
I cannot reproduce it here:

```
D:\test> live-server                                                                                                                                                                                    2024/08/06 23:35:11
[2024-08-06T15:35:16Z INFO  live_server::server] Listening on http://192.168.31.113:8860/
[2024-08-06T15:35:16Z INFO  live_server::watcher] Listening on \\?\D:\test
[2024-08-06T15:35:21Z WARN  live_server::server] Failed to read "\\?\D:\test\favicon.ico": 系统找不到指定的文件。 (os error 2)
```

![image](https://github.com/user-attachments/assets/3ee20339-d82b-4f7e-a173-4b717f00f8a8)

**Comment by doronbehar** [comment]
> I cannot reproduce it here:

Very peculiar! I just tested it again also with chromium and same issue. However, testing the latest version from Git seemed to fix the issue! The stable release that has this issue that I'm using here on NixOS is 0.7.0. Is there a chance for a new release please? :pray:

**Comment by lomirus** [comment]
Yes, it's a good chance to release a new version since the latest release from last year after 8 months. I will release a new version in the next few days (or maybe on the weekend at least).

Sent from Proton Mail Android

-------- Original Message --------

**Comment by lomirus** [comment]
Closed as v0.8.0 has just released

**Comment by doronbehar** [comment]
Thanks!
